dashdblocal_notebooks/livy-notebooks/IoT_Demo_Pyspark.ipynb

```diff
@@ -0,0 +1,1246 @@
BCCpOmZp3HHXQgIMPZtTPfso+cyopP+kkKCpiw72/49mzzuLZj5xNze9/T96n2lPvkGuxMSK+HRETImJMRBwUEf8JHAP8utU+44CjgF/kFFOSJEmSJBVUVc4BYNC7303pHnvknEad1X+//djrB5cwdt7tDD3jdCgtpfbPf+b5c89j+fs/wPoFC0jNzXnHVA+Wd2fj7sD1ZOdtXAgcDkxOKc1vtc/HgBeBu3d+PEmSJEmS1CI1N1NVWIW6YsaMnNNoe/QbM4aR3/wm4+bfzbCzP0L078+mv/2NFz75rzwzbRpVc+eSGhvzjqkeKPfVqHe0iCgHqlyNWpIkSZKk7bPhj3/iuZkzKRo8mPH3LaGof/+8I6mLNK5Zw5rrfsnaX/+a5poaAEpHjWLEeedRMWM6Rf365ZxQeeppq1FLkiRJkqQeoKqwMEz5lCkWGnuZkuHD2e3TFzHunkXsetFFFA8bRsPzz7Piq19l2aTjWX3ttTTX1uYdUz2AnY2SJEmSJGmrmjds4MkJR5Fqa3nTDb9m4DvekXck7UDNGzey7pZbWH3V1TS+8goAxUOHMuzsjzD8Qx+iOOtyUx9hZ6MkSZIkSepS1XfPJ9XWUvqm0Qw45JC842gHKxowgOFnn83Y+Xezxze/Qeno0TStW8eqH/8PS4+byMof/IDGVavyjqluyGKjJEmSJEnaqpaFYYZOn05E5JxGO0tRv34MO+MMxs67nT0vuYSy8eNp3rCB1Vf+gqUTJ7Him9+i4aWX8o6pbsQxakmSJEmStEX1L7zIskmTIIJxCxdQuueeeUdSTlJzMzWLF7PqssvZ9Mgj2caSEipOOYUR559H2T775BtQO4Rj1JIkSZIkqctUzcm6GgcefriFxj8LPTkAACAASURBVD4uiooYctxxjLn5JkZfczUDjzgCGhup+u1vefrEk3jh059m0z/+kXdM5chioyRJkiRJ2qyUElWVcwAYOmN6zmnUXUQEg448kjddew1jbrqRwcceCymx/o47eWb6DJ6f9XFqH34475jKgWPUkiRJkiRps2offJBnP/wRigYOZPx9SygaODDvSOqmNj3xBKsvv4LqO++E5mYABh52GLt8fBYDjzzSc332YI5RS5IkSZKkLrGusDDMkMmTLTRqi/rvtx97XfoDxs67nYrTT4PSUmr/9Cee+9i5LP/AmaxfuJBUKEKq97KzUZIkSZIktat540aeeu8Emjds4E3X/5KBhx6adyT1IA0vv8zqq69h3S23kDZtAqBs/HhGXHAB5VMmEyUlOSdUR9nZKEmSJEmSttv6BQto3rCB0r33ZsA735l3HPUwpSNHsscXv8C4hQsYccEFFA0eTN1TT/HS5z7Hsiknsvbm39BcX593THUxi42SJEmSJKldVbNnA1AxfTpRZAlB26ZkxAh2+8ynGbdoIbtedCHFw4bR8PzzrPjqV1k26XhWX3stzbW1ecdUF3GMWpIkSZIk/ZOGl19m6XETISXGLphPv733zjuSeonm2lrW3XILq6++hsZXXgGgeOhQhs88m2Ef+hDF1m+6HceoJUmSJEnSdqmacyukxMBDD7XQqC5VNHAgw2fOZOz8u9njm9+gdPRomtat49X//jFLjz2OlT+4lMbVq/OOqW1ksVGSJEmSJL1BSukNI9TSjlDUrx/DzjiDsfNuZ89LLqFs/HiaN2xg9ZVXsvS4iaz41n/S8PLLecdUJzlGLUmSJEmS3qD24Yd59oNnEQMGMH7JEooHD8o7kvqA1NxMzeLFrLrscjY98ki2sbSUilOmssv559NvzJhc8/VljlFLkiRJkqRtVjW7EoDy973PQqN2migqYshxxzHm5psYfc3VDDz8cGhooOr//ZZlJ57Ei5/5DJueeCLvmNoKOxslSZIkSdJrmjdt4qkJR9G8fj2jr72GQUcckXck9WG1Dz/M6suvoGbx4te2DT7mGEbMuoCBhxySX7A+xs5GSZIkSZK0TdYvXEjz+vWU7DmSgYcdlncc9XEDDzmEUZf9nH0qZ1N+4hSIoGbxYp794Fk8O/McNvzhD/T2RrqexmKjJEmSJEl6TVXlHAAqpk0jiiwbqHvov//+7HXppew773YqTjsVSkqo/eMfee5j57L8A2eyfuFCUnNz3jGFY9SSJEmSJKmg4ZWVLD32WGhuZuxdd9LvTW/KO5LUroaXXmL11dew7pZbSHV1AJSNH8+ICy6gfMpkoqQk54S9i2PUkiRJkiSp06rn3grNzQx4xzssNKpbK91zT/b40hcZt2ghIy64gKLBg6l76ile+tznWHbiSaz9zW9orq/PO2afZLFRkiRJkiSRUmJdYRXqihnTc04jdUzJiBHs9plPM27RQna96EKKhw6l4bnnWPGVr7Ls+Pex5rrraK6tzTtmn+IYtSRJkiRJYuMjj7D8/R8gysoYf98SiocMyTuS1GnNtbWsu+UWVl91NY0rVwJQPGwYw2eezbCzzqLY2tA2cYxakiRJkiR1SlVl1tU45PjjLTSqxyoaOJDhM2cydsF89vjG1ykdNYqmtWt59Uf/zdLjJrLy0h/SuHp13jF7NTsbJUmSJEnq45rr63lqwlE0V1Ux6qpfMPg978k7ktQlUmMj1XfcyeorLqfuqaUARP/+DD3jDEZ87KOUjhyZc8Kewc5GSZIkSZLUYTWL7qG5qoqS3Xdn0BFH5B1H6jJRUkLF1JPZZ84c9v7pT+h/0EGkTZtYe/31LH3fCbz0pS9Rv3x53jF7FYuNkiRJkiT1cVWzZwNQMW0aUVyccxqp60VREUMmTmTMb25m9NVXMfCww6Chgar//X8sO/EkXvzMv7HpiSfyjtkrOEYtSZIkSVIf1vjqqzx1zLHQ1MS+8+ZRtu8+eUeSdorahx5m9eWXU3Pvva9tG3zssewy6wIGHHxwjsm6H8eoJUmSJElSh1TNvQ2amhjw9rdbaFSfMvAdhzDq8svYZ/ZvKT9xCkRQc889LD/zgzx7zkfZcP/99PYmvR3BYqMkSZI
```

Comment:
I'm not a lawyer, but https://github.com/ibmdbanalytics/dashdb_analytic_tools/blob/master/dashdblocal_notebooks/LICENSE contains the 3-clause BSD license.

Reply:
ok it may be safer to write 'All rights reserved' then